cpu/em_quad_level_frequency_transitions.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+#
+# Copyright: 2018 IBM
+# Author: Shriya Kulkarni <shriyak@linux.vnet.ibm.com>
+import os
+import time
+import random
+import platform
+from avocado import Test
+from avocado import main
+from avocado.utils import process, distro, cpu
+from avocado import skipIf
+from avocado.utils.software_manager import SoftwareManager
+
+IS_POWER_NV = 'POWER8' in open('/proc/cpuinfo', 'r').read()
+
+
+class freq_transitions(Test):
+    """
+    To validate quad level frequency transitions.
+    """
+    @skipIf(IS_POWER_NV, "This test is not supported on Power 8 platform")
```

Comment:
what about older Power Platform ?

Reply:
This is supported only on P9 and onwards. Let me make changes accordingly

Reply:
sure